content/project-adopters/iot.md

```diff
@@ -2,4 +2,6 @@
 title: "IoT Project Adopters"
 date: 2019-09-10T15:50:25-04:00
 working_group: internet-things-iot
----
\ No newline at end of file
+---
+
+{{< eclipsefdn_adopters working_group="internet-things-iot" >}}
```

Comment:
If these pages are updated, we should update all pages, not just 2 (this is why it was in the page template, to reduce repetition)